answers/javaScript.md

```diff
@@ -1,5 +1,6 @@
 ## JavaScript
 * `null` vs `undefined` - Null means there is 'no value.' Undefined represents a variable that has not been declared or not given a value.
 	* What does `type of null` output? It returns 'object.'
-* `==` vs `===` - == compares values but not data types. eg. 3 == "3" would be true. ie... it converts the values.
-=== compares values including data types. 3 === "3" would be false ie.... it doesn't convert values.
+* `==` vs `===`
+		- == compares values but not data types. eg. 3 == "3" would be true. ie... it converts the values.
+		-	=== compares values including data types. 3 === "3" would be false ie.... it doesn't convert values.
```

Comment:
@kegillett put anything that represents code inside "`" (backticks)

Reply:
so `===` or `3 === "3"`

Reply:
fixed, noted and will push again, thanks Jack.